src/components/SearchBar/SearchBar.jsx

```diff
@@ -1,17 +1,18 @@
-import React, {Component} from 'react';
+import React from 'react';
+import SearchBy from '../SearchBy/SearchBy';
 import styles from './search-bar.scss';
 
-class SearchBar extends Component {
-  render() {
-    return (
-      <form>
-        <h1>FIND YOUR MOVIE</h1>
-        <input type="email" id="search-movie" placeholder="Search" />
-        <p className="text-uppercase">Search by: <a href="#">Director</a> <a href="#">Title</a></p>
-        <button type="submit">Search</button>
-      </form>
-    );
-  }
-}
+const SearchBar = () => (
+  <div className={styles.searchForm}>
+    <form>
+      <p className={styles.formTitle}>Find your movie</p>
+      <input type="email" id={styles.searchBar} placeholder="Search" />
+      <SearchBy />
```

Comment:
Could you please pass callback to SearchBy? It should change state of the SearchBar and put current search filter (director or title)